scaffold.yaml

```diff
@@ -59,29 +60,36 @@ features:
     globs:
       - "*/package.json"
       - "*/pnpm-*.yaml"
-      - "*/.eslintrc*"
+      - "*/eslint.config.mjs*"
```

Comment:
is this related to go?

Reply:
no sorry, it was a drive-by cleanup from the eslint upgrade where I missed this, and caused non-JS projects to get this file.